src/snowflake/connector/connection.py

```diff
@@ -145,11 +146,13 @@ def DefaultConverterClass() -> type:
     "schema": (None, (type(None), str)),  # snowflake
     "role": (None, (type(None), str)),  # snowflake
     "session_id": (None, (type(None), str)),  # snowflake
-    "login_timeout": (120, int),  # login timeout
+    "login_timeout": (None, (type(None), int)),  # login timeout
```

Comment:
what will be default timeout if we change this to None?

Reply:
In `by_plugin` I defined `DEFAULT_AUTH_CLASS_TIMEOUT = 120`. I though it made the arguments to `connect` a bit more consistent.